export function handleRejection(e: Error): void {
  // catch promise rejection to exception handler
  throw e;
}
